### DISCUSSION TITLE
option to install pip packages

### DISCUSSION THREAD

**Discussion by Zoey2936** [nextcloud/all-in-one#2071] [Ideas] [answered]
The yt-dlp version in alpine v3.16 is currently broken and v3.17 is currently blocked (#1846), so it would be nice to have the ability to install pip packages, I would like to create a PR, but before I wanted to ask if this option would be accepted? @szaimen

**Comment by szaimen** [comment]
> The yt-dlp version in alpine v3.16 is currently broken

Why cant they fix this?

**Reply by Zoey2936** [reply]
the fix is only aviable in v3.17

**Reply by Zoey2936** [reply]
(and pip)

**Reply by szaimen** [reply]
And why cannot they backport the fix? I thought alpine 3.16 would still be supported for many years or am I wrong?

**Reply by Zoey2936** [reply]
No idea, v3.17 got the package fixed on the same day as the fix was released and v3.16 has a yt-dlp version from early 2022

**Reply by szaimen** [reply]
Is there a way to ask them to update the package also for 3.16?

**Reply by Zoey2936** [reply]
Maybe here: https://gitlab.alpinelinux.org/alpine/aports/-/issues
But I'm not sure if they would do it

**Reply by Zoey2936** [reply]
v3.17 got this fixed: https://gitlab.alpinelinux.org/alpine/aports/-/commit/aa1c2f4bd17dbd627f7120a1cd56bb18ab912672
v3.16 last yt-dlp update was 9 months ago: https://gitlab.alpinelinux.org/alpine/aports/-/commit/1447516369c13f517498eed3d57071cba909a78f

**Reply by szaimen** [reply]
Can one ask for this in their issue tracker?

**Reply by Zoey2936** [reply]
It feels wired to open an Issue, which asked if a package can be moved from edge to v3.17 and at the same time opening an Issue, which asks for an updated Version of a package inside v3.16

**Reply by Zoey2936** [reply]
They won't fix it: https://www.alpinelinux.org/releases v3.16 only receives security fixes, no bug fixes

**Reply by Zoey2936** [reply]
and the yt-dlp is broken, because of a change on YouTube, so not a security problem

**Reply by szaimen** [reply]
Looks like bad luck then :/

**Comment by szaimen** [comment, marked as answer]
Honestly I dont want to add support for this since usually all dependencies are available via apk and/or php-ext. Since this currently seems to be the only dependency not working, I rather want to fix this by updating the alpine image to 3.17 which will be done together with the update to nc26

**Reply by Zoey2936** [reply]
Then I will use until then invidious
